kodilla-rps/src/main/java/InputtingData.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+
+public class InputtingData {
+
+    Scanner scanner = new Scanner(System.in);
+
+    public String introduceText() {
```

Comment:
I think that better names will be readText, readNumber...

Reply:
done